conans/client/conan_api.py

```diff
@@ -962,13 +963,21 @@ def remote_clean(self):
         return self._cache.registry.clear()
 
     @api_method
-    def remove_locks(self):
-        self._cache.remove_locks()
+    def remove_system_reqs(self, ref):
+        try:
+            self._cache.package_layout(ref).remove_system_reqs()
+            self._user_io.out.info(
+                "Cache system_reqs from %s has been removed" % repr(ref))
+        except Exception as error:
+            raise ConanException("Unable to remove system_reqs: %s" % error)
 
     @api_method
-    def remove_system_reqs(self, reference):
-        ref = ConanFileReference.loads(reference)
-        self._cache.package_layout(ref).remove_system_reqs()
+    def remove_system_reqs_by_pattern(self, pattern):
+        for ref in search_recipes(self._cache, pattern=pattern):
+            self.remove_system_reqs(ref)
+
+    def remove_locks(self):
+        self._cache.remove_locks()
```

Comment:
Oops! It looks like the decorator has gone in the merge:

```python
@api_method
def remove_locks(self):
    self._cache.remove_locks()
```

Reply:
My bad

Reply:
Thanks! I'm sure it has been the merge 😄